docs/federation.md

```
@@ -0,0 +1,213 @@
+```bash
+npx defense-in-depth verify
+```
+
+If the parent ticket is in a blocked phase, the federation guard will emit a
+BLOCK finding and prevent the commit.
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)